cypress/package.json

```diff
@@ -20,5 +23,10 @@
   },
   "engines": {
     "node": "^12.22 || ^14.13 || >=16"
+  },
+  "scripts": {
+    "delete:reports": "rm cypress/results/* || true",
+    "prereport": "yarn run delete:reports",
```

Comment:
`yarn` is not required by default, we can use `npm run delete:reports` instead.

Reply:
thanks!